src/nvim/fileio.c

```diff
@@ -366,14 +366,13 @@ readfile (
   /* don't display the file info for another buffer now */
   need_fileinfo = FALSE;
 
-  /*
-   * For Unix: Use the short file name whenever possible.
-   * Avoids problems with networks and when directory names are changed.
-   * Don't do this for MS-DOS, a "cd" in a sub-shell may have moved us to
```

Comment:
I would assume so yes but I did not look into it. AFAIK cd will change the current directory.

However if you look at the Vim source `BACKSLASH_IN_FILENAME` is defined in `os_dos.h` which gets included in `os_win32.h` so I would assume we want this same behavior.

Just FYI we do not define `BACKSLASH_IN_FILENAME` yet for Windows but we should AFAIK.